Source/Models/BrickDimension.swift

```diff
@@ -131,6 +131,13 @@ public enum BrickDimension {
         }
     }
 
+    func isRatio() -> Bool {
+        if case BrickDimension.ratio = self {
```

Comment:
Does this also work with `indirect` BrickDimensions? Because it could be a `orientation(landscape: BrickDimension, portrait: BrickDimension)` or others that result in being a ratio, but not at first. You can check that by calling `self.dimension(withValue: nil)`, which should trickle down until it has a `direct` Dimension
Adding a test with an indirect dimension would be recommended

Reply:
Sounds good! I'll update with those changes.